web/api/v1/api.go

```diff
@@ -147,6 +148,8 @@ func (api *API) Register(r *route.Router) {
 
 	r.Get("/targets", instr("targets", api.targets))
 	r.Get("/alertmanagers", instr("alertmanagers", api.alertmanagers))
+
+	r.Get("/read", prometheus.InstrumentHandler("read", http.HandlerFunc(api.remoteRead)))
```

Comment:
So I'll leave it where it is?  I don't intend on adding this endpoint in v2 yet, I'd like to see it as gRPC like the other APIs there.